enhancements/network/ovn-hypershift-live-migration.md

```diff
@@ -0,0 +1,295 @@
+---
+title: OVN-kubernetes live migration for hypershift kubevirt provider
+authors:
+  - @ellorent
+reviewers:
+  - "@dcbw"
+  - @danwinship
+  - @tssurya
+  - @numans
+approvers:
+  - TBD
+creation-date: 2023-02-06
+last-updated: 2023-02-06
+tracking-link: 
+  - "https://issues.redhat.com/browse/CNV-22946"
+see-also:
+  - "https://hypershift-docs.netlify.app/"
+---
+
+
+# OVN-kubernetes live migration for hypershift kubevirt provider
+
+## Summary
+
+The [Hypershift](https://hypershift-docs.netlify.app/) project creates openshift clusters on demand on top of already provisioned openshift by deploying "Hosted Control Planes" and attach worker nodes to "Hosted Control Planes" using oneof different providers, such as aws, azure, baremetal (agent), and kubevirt, in the case of kubevirt it will spins up the workers as kubevirt virtual machines.
+
+There are some network requirements, some of them are imposed by kubevirt 
+live migration and others by kubernetes:
+
+- Consistent IP follows VM during Live Migration, and is the IP of the primary guest node interface that the kubelet uses to register/communicate with.
```

Comment:
While I understand the sentence, I think it may be a bit convoluted.

How about we focus on the kubelet IP ? Something like:
> The IP Kubelet is listening to follows the node during live migration

Just a suggestion; while it is simpler, I'm unsure it reads much better.